src/dto.ts

```diff
@@ -2,6 +2,9 @@ import { ApiProperty } from '@nestjs/swagger';
 
 export const contractPattern = 'ct_([1-9a-zA-Z]){49,50}';
 export const bigNumberPattern = '[1-9]+';
+export const microBlockTimePattern = '[1-9]{13}';
+export const microBlockHashPattern = 'mh_([1-9a-zA-Z]){49,50}';
```

Comment:
i think the are base56 encoded, so the alphabet is `23456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnpqrstuvwxyz`, meaning it can not contain 1, i, I, o, O,  or 0